app/assets/javascripts/local_time.js.coffee

```diff
@@ -200,7 +200,8 @@ update = (callback) ->
   document.addEventListener "time:elapse", callback
 
   if Turbolinks?.supported
-    document.addEventListener "page:update", callback
+    for event in ["page:update", "turbolinks:load"]
+      document.addEventListener event, callback
```

Comment:
Not exactly related to this change, but simply checking `Turbolinks.supported` will miss the jQuery `ajaxSuccess` event when using TL5. TL classic [triggers `page:update` on `ajaxSuccess`](https://github.com/turbolinks/turbolinks-classic/blob/master/lib/assets/javascripts/turbolinks.coffee#L654-L656) and TL5 does not (unless you include the [compatibility shim](https://github.com/turbolinks/turbolinks/blob/master/src/turbolinks/compatibility.coffee)).

So the ideal condition, in somewhat pseudo code, would be:
```coffee
if Turbolinks?.supported
  if Turbolinks.version < 5 or Turbolinks.version >= 5 and Turbolinks.compatibilityMode
    # Add page:update listener
  else
    # Add turbolinks:load listener
    # Add jQuery ajaxSuccess listener
else
  # Add jQuery ajaxSuccess listener
```
/cc @sstephenson @packagethief